documentation/changelog.rst

```diff
@@ -14,6 +14,7 @@ Infrastructure / Support
 
 * New documentation section on constructing a flex model for :abbr:`V2G (vehicle-to-grid)` [see `PR #885 <https://github.com/FlexMeasures/flexmeasures/pull/885>`_]
 * Allow charts in plugins to show currency codes (such as EUR) as currency symbols (€) [see `PR #922 <https://github.com/FlexMeasures/flexmeasures/pull/922>`_]
+* Expose device-level power constraints as sensors via de API [see `PR #897 <https://github.com/FlexMeasures/flexmeasures/pull/897>`_]
```

Comment:
de -> the

I also like the changelog entries (especially the features) to signal how they may be valuable for users, while retaining brevity. Perhaps our PR template should include such a comment.

In this case, I also have a hard time making it not too technical. There's user value in being able to schedule against constraints that change within the planning horizon, while retaining a history of those constraints (as sensor data), which I think is the main feature of this PR. We just happened to tackle device power constraints first. But let's not dwell on it, as we should have ample opportunity to make our point later, upon exposing this functionality in other API fields.

Reply:
You're right, this should put more emphasis on the impact for the end-users. 

I've updated the entry trying to capture the essence.

Feel free to propose an alternative  :smile: .

Reply:
I think you nailed it now.